2 new segments: show just the file's name & show path AND file name

MikeTheGreat opened the pull request:
These were useful for me, so I'd like to share them with y'all.

Please let me know what I can do to help with this process!

dbordak commented:
You have a name collision with the other `buffer-name` segment. Yours is probably the better implementation, though -- that variable has a mouse map, right? No reason not to include that :P

Anyway, the other one if I understand correctly is always going to print the path from root? If that is the case, you should probably name it `absolute-path` or something along those lines.

MikeTheGreat commented:
Name collision: D'oh!!! It was _right_above_ the one I added.  I'm sorry, and feel embarrassed that I did that.

Ok, I've added another commit ( 1025d43 ) which removes the `buffer-name` segment that was there originally, and I've renamed the other one to include  `absolute-path` in the name.

Is there anything else I should/can do?  (I've never updated a PR before :)  )
